configure-ironic.sh

```diff
@@ -2,6 +2,14 @@
 
 . /bin/ironic-common.sh
 
+USE_HTTP_BASIC=${USE_HTTP_BASIC:-false}
+IRONIC_HTTP_BASIC_FILE=${IRONIC_HTTP_BASIC_FILE:-"/shared/htpasswd-ironic"}
```

Comment:
We don't need the file to be provided, we can generate it.

Oh, I see you do generate it. So we don't need an option, let's be opinionated here.

Reply:
so we shouldn't provide a way to be customizable? the value we need to set ``http_basic_auth_user_file``

Reply:
If the file is generated within the container and consumed by the container, why does it need to be configurable to the user?